recipes/pannb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pannb" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pannb-{{ version }}.tar.gz
+  sha256: 7df407f4c15173648dab99a9252c551274c76f24dd9e812687aaa2a60dd65a36
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pannb = pannb:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - poetry >=1,<2
+  run:
+    - panflute >=2.1,<3.0
+    - python >=3.8
+  run_constrained:
+    - pandoc >=2.11.2,<2.15
+    - coloredlogs >=14,<16
+
+test:
+  imports:
+    - pannb
+  commands:
+    - pip check
+    - |
+      echo '# testing' | pandoc -F pannb
+  requires:
+    - pip
+    - pandoc >=2.11.2,<2.15
+
+about:
+  home: https://github.com/ickc/pannb/
+  summary: pannb—pandoc filter for ipynb
```

Comment:
A `description` field with some more info would be nice.

Reply:
I don't know when this becomes a requirement. I created around 50 feedstocks and this has never been a requirement. Most of the time the upstream hasn't even put a description in their `setup.py` or equiv.

Why do you think this is suddenly necessary here?

Reply:
It's not a requirement, just nice to have. No need to add it.

Reply:
Unless it is a requirement, it is extra friction to add that. Consider yourself as a maintainer of many things too, an extra unnecessary input coupled with maintaining dozens of things will becomes a huge burden.